pizzaplace/services/pizza_place_presenter.py

```diff
@@ -0,0 +1,19 @@
+class PizzaPlacePresenter(): 
+  def __init__(self, pizza_place, YelpApi):
+    self.pizza_place = pizza_place
+    self.yelp_api = YelpApi
+
+  def get_yelp_link(self):
+    return self.pizza_place.yelp_link
+
+  def get_pizza_place_name(self):
+    return self.pizza_place.name
+
+  def get_pizza_place_rating(self):
+    response = self.yelp_api(self.get_yelp_link()).get_response()
```

Comment:
The same criticisms of the `MeetupPresenter` apply here, too: Consider adding another layer responsible for coordinating the logic that gets an API response and parses out the details this presenter cares about.